src/components/SelectField/SelectField.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+import { Select } from './style';
+
+function SelectField(props) {
+  const {
+    error, onChange, options, defaultText,
+  } = props;
+  return (
+    <>
+      <Select onChange={onChange} error={error}>
+        { defaultText && <option>{defaultText}</option>}
+        {
+          options && options.length
+          && options.map(({ value, label }) => <option key={label} value={value}>{label}</option>)
```

Comment:
use default blank array in default propTypes for options field and remove the && conditions